airflow_bridge/entrypoint.sh

```diff
@@ -71,9 +71,8 @@ fi
 
 case "$1" in
   webserver)
-    airflow db reset
-    airflow db init
     airflow db upgrade
+    airflow db init
```

Comment:
why init after upgrade?

Reply:
https://github.com/apache/airflow/blob/main/airflow/utils/db.py#L592-L606 seems like these steps happen after upgrade

Reply:
looking back at this db init calls upgrade already, so I could probably get rid of db upgrade command line